src/functions/template-tags/date.php

```diff
@@ -359,7 +373,7 @@ function tribe_get_end_date( $event = null, $display_time = true, $date_format =
 		 * @param string  $end_date
 		 * @param WP_Post $event
 		 */
-		return apply_filters( 'tribe_get_end_date', tribe_format_date( $end_date, $display_time, $date_format ), $event );
+		return $end_dates[ $cache_key ] = apply_filters( 'tribe_get_end_date', tribe_format_date( $end_date, $display_time, $date_format ), $event );
```

Comment:
:no_entry_sign: Assignments must be the first block of code on a line

<sub>hash: 8af02b7558368c1c3e695b392990310e</sub>